packages/web-app-files/src/mixins/actions/emptyTrashBin.js

```diff
@@ -27,12 +32,35 @@ export default {
     }
   },
   methods: {
-    ...mapActions(['showMessage']),
+    ...mapActions(['showMessage', 'createModal', 'hideModal', 'toggleModalConfirmButton']),
     ...mapActions('Files', ['clearTrashBin']),
+
     $_emptyTrashBin_trigger() {
-      this.$client.fileTrash
-        .clearTrashBin()
+      const modal = {
+        variation: 'danger',
+        title: this.$gettext('Empty trash'),
```

Comment:
```suggestion
        title: this.$gettext('Empty trash bin'),
```

Reply:
Not sure, see linked ticket @tbsbdr suggestions?

Reply:
👍 "trash bin" in this case, thx